### ISSUE TITLE
No terminal matches "#" in the current parser context

### ISSUE THREAD

**Issue by oysteoh** [equinor/ert#5354]
From log messages this one shows up after the new release. Is this something we need to fix?

```
The new parser failed
Traceback (most recent call last):
  File "/prog/res/komodo/2023.05.00-py38-rhel7/root/lib64/python3.8/site-packages/ert/_c_wrappers/enkf/lark_parser.py", line 303, in _parse_file
    tree = _parser.parse(content + "\n")
  File "/prog/res/komodo/2023.05.00-py38-rhel7/root/lib/python3.8/site-packages/lark/lark.py", line 645, in parse
    return self.parser.parse(text, start=start, on_error=on_error)
  File "/prog/res/komodo/2023.05.00-py38-rhel7/root/lib/python3.8/site-packages/lark/parser_frontends.py", line 96, in parse
    return self.parser.parse(stream, chosen_start, **kw)
  File "/prog/res/komodo/2023.05.00-py38-rhel7/root/lib/python3.8/site-packages/lark/parsers/earley.py", line 266, in parse
    to_scan = self._parse(lexer, columns, to_scan, start_symbol)
  File "/prog/res/komodo/2023.05.00-py38-rhel7/root/lib/python3.8/site-packages/lark/parsers/xearley.py", line 146, in _parse
    to_scan = scan(i, to_scan)
  File "/prog/res/komodo/2023.05.00-py38-rhel7/root/lib/python3.8/site-packages/lark/parsers/xearley.py", line 119, in scan
    raise UnexpectedCharacters(stream, i, text_line, text_column, {item.expect.name for item in to_scan},
lark.exceptions.UnexpectedCharacters: No terminal matches '#' in the current parser context, at line 350 col 126

NV-<IENS>.DATA, <INSERT_THIS>=nlCPRnl/nl#, <SPACE_DELIMITER>=#, <NEWLINE_ID>=nl)
                                        ^
Expected one of: 
	* RPAR
	* COMMA
```

**Comment by DanSava** [comment]
I could not reproduce using main that has https://github.com/equinor/ert/pull/5327 merged

Tested it locally and it seems the changes from https://github.com/equinor/ert/pull/5327 fixes the issue also

**Comment by oysteoh** [comment]
Resolved by #5327
